custom_components/tahoma/cover.py

```diff
@@ -121,9 +121,13 @@ def update(self):
             )
             
         if CORE_TARGET_CLOSURE_STATE in self.tahoma_device.active_states:
-            self._position = 100 - self.tahoma_device.active_states.get(
+            self._position = self.tahoma_device.active_states.get(
                 CORE_TARGET_CLOSURE_STATE
             )
+            # Fix for unknown position after moving with remote control
+            # and closure states over 100
+            if self._position > 0:
```

Comment:
I am not sure when this could happen. The value that TaHoma returns is always between 0 and 100 I believe? Could we fix the other code to make sure don't end below 0 or above 100?

Reply:
I comment this in the other conversation and push the current solution.